graylog2-web-interface/src/components/event-definitions/event-definition-form/FieldsForm.tsx

```diff
@@ -111,7 +113,7 @@ const FieldsForm = ({ currentUser, eventDefinition, validation, onChange }: Prop
       <Col md={12}>
         <h2 className={commonStyles.title}>Event Fields <small>(optional)</small></h2>
 
-        {isSystemEventDefinition ? (
+        {!canEditCondition ? (
           <p>
             The event fields of system notification event definitions cannot be edited.
```

Comment:
These changes allow other types of event definitions to reach this component too, so we might want to make this statement slightly more generic. Open to different wording but here's a suggestion
```suggestion
            The event fields of this event definition type cannot be edited.
```